src/private/m2kcalibration_impl.cpp

```diff
@@ -776,6 +778,13 @@ class M2kCalibration::M2kCalibrationImpl {
 		return true;
 	}
 
+	int16_t processRawSample(int16_t value)
+	{
+		// This should go away once channel type gets
+		// changed from 'le:S16/16>>0' to 'le:S12/16>>4'
+		return ((-value) << 4);
```

Comment:
is this a driver change? or a change that should be done in the driver?
if yes, maybe we can do it now and use that as base

Reply:
I don't have a lot of details about this change. We can investigate it further.

Reply:
@mhennerich 
any thoughts/input here?

Reply:
All our DAC HDL cores expect the realbit MSB aligned with the MSB of the storage bit.
Even it the change the type, which is basically just a string, I don't think anything would go away here...

Reply:
then should we remove the string as it may be misleading?
or change it to something else?

Reply:
Ack - change comment to:
12-bit DAC values must be 16-bit MSB aligned.

Reply:
Comment updated